### PR TITLE
Chore/link to circleci job

### PR DESCRIPTION
**Pull request by Shesekino**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)
- [x] Potential release notes have been inspected

### What this does

link to the failed CircleCI job on failures

### More information

https://snyk.slack.com/archives/CLW30N31V/p1583868837007600

### CODE REVIEW

File: scripts/slack-notify-failure.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@
 BRANCH_NAME="$1"
 NOTIFICATION_COLOR=${2:-#EE0000}
 
-curl -X POST -H 'Content-Type:application/json' -d '{"attachments": [{"color": "'$NOTIFICATION_COLOR'", "fallback": "Build Notification: '$CIRCLE_BUILD_URL'", "title": ":warning: Kubernetes-Monitor Merge Failure :warning:", "text": ":egg_broken_1: Kubernetes-Monitor broken branch: `'$BRANCH_NAME'` :egg_broken_1:"}]}' $SLACK_WEBHOOK
+curl -X POST -H 'Content-Type:application/json' -d '{"attachments": [{"color": "'$NOTIFICATION_COLOR'", "fallback": "Build Notification: '$CIRCLE_BUILD_URL'", "title": ":warning: Kubernetes-Monitor Merge Failure :warning:", "text": ":egg_broken_1: Kubernetes-Monitor broken branch: `'$BRANCH_NAME'`:egg_broken_1:\n'$CIRCLE_BUILD_URL'"}]}' $SLACK_WEBHOOK
```
Comment (ivanstanev) [inline scripts/slack-notify-failure.sh]: Any reason to remove the extra space?
```shell
`'$BRANCH_NAME'` :egg_broken_1:"
```
I think it looks symmetric to the space at the start of the message 😛

Comment (Shesekino) [reply scripts/slack-notify-failure.sh]: yes, the answer is in the question's body: it's an extra space
it does nothing, and is followed by a newline
why would we want it? 😬

Comment (Shesekino) [reply scripts/slack-notify-failure.sh]: oh I think I got you

Comment (Shesekino) [reply scripts/slack-notify-failure.sh]: yeah 🦅 👁 it looks cleaner with the space! sorry

Comment (Shesekino) [reply scripts/slack-notify-failure.sh]: check again please

**Pull request merged**